yb-voyager/cmd/root.go

```diff
@@ -66,29 +66,46 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 		if !shouldRunPersistentPreRun(cmd) {
 			return
 		}
-		validateExportDirFlag()
-		schemaDir = filepath.Join(exportDir, "schema")
-		if shouldLock(cmd) {
-			lockFPath := filepath.Join(exportDir, fmt.Sprintf(".%sLockfile.lck", GetCommandID(cmd)))
-			lockFile = lockfile.NewLockfile(lockFPath)
-			lockFile.Lock()
-		}
-		InitLogging(exportDir, cmd.Use == "status", GetCommandID(cmd))
-		startTime = time.Now()
-
-		if callhome.SendDiagnostics {
-			createCLIArgsString(cmd)
-			go sendCallhomePayloadAtIntervals()
-		}
 
-		log.Infof("Start time: %s\n", startTime)
-		if metaDBIsCreated(exportDir) {
-			initMetaDB()
-		}
-		if perfProfile {
-			go startPprofServer()
+		if currentCommand == "yb-voyager assess-migration-bulk" {
```

Comment:
I think here we can have separate function for bulk commands identification (right now it will have only one but maybe future we plan to add more we can just it to that)
`isBulkCommand(cmd)`.